app/scripts/HiGlassComponent.js

```diff
@@ -3255,16 +3296,28 @@ class HiGlassComponent extends React.Component {
       relTrackY: (hoveredTrack && hoveredTrack.flipText) ? relTrackPos[0] : relTrackPos[1],
       dataX,
       dataY,
-      isFrom2dTrack: hoveredTrack && hoveredTrack.is2d,
-      isFromVerticalTrack: hoveredTrack && hoveredTrack.flipText,
+      isFrom2dTrack: !!(hoveredTrack && hoveredTrack.is2d),
```

Comment:
Same comment about `noHoveredTracks` below. Derived boolean variables just add clutter, IMO.

Reply:
Same as above. If `hoveredTrack` was available in another window we wouldn't need it but that array is not available so we require those derived boolean variables.